src/code/z_play.c

```diff
@@ -1,3 +1,4 @@
+#pragma increment_block_number "gc-eu:16 gc-eu-mq:16 gc-jp:16 gc-jp-ce:16 gc-jp-mq:16 gc-us:16 gc-us-mq:16"
```

Comment:
it's arguably better to put the `#pragma increment_block_number` before the headers, but we haven't been doing that and I'd like to stay consistent. Hopefully the same increments just work